src/Profile/ProfileScreen.js

```diff
@@ -105,60 +114,53 @@ const airtableConfig = {
 const base = new Airtable({ apiKey: airtableConfig.apiKey })
   .base(airtableConfig.baseKey);
 
-// eslint-disable-next-line no-unused-vars
 export default function ProfileScreen({ navigation }) {
-  const DUMMY_USER_ID = 'rec0hmO4UPOvtI3vA';
-  const DUMMY_NAME = 'Joe Bruin';
-
-  const [email, setEmail] = useState('');
-  const [phoneNum, setPhoneNum] = useState('');
-  const [address, setAddress] = useState('');
+  const currentUser = store.getState().auth.user;
 
-  const [avatar, setAvatar] = useState(require('../assets/imgs/placeholder.png'));
+  const [title, setTitle] = useState('Edit');
+  const [isEditMode, setEditMode] = useState(false);
+  const [refresh, setRefresh] = useState(0);
 
-  useEffect(() => {
-    const useremail = 'helen@gmail.com';
-    base('Users').select({
-      filterByFormula: `({email}='${useremail}')`,
-    }).firstPage().then((record) => {
-      switch (record[0].fields.avatarNum) {
-        case 1: setAvatar(require('../assets/imgs/pipa.png'));
-          break;
-        case 2: setAvatar(require('../assets/imgs/eggplant.png'));
-          break;
-        case 3: setAvatar(require('../assets/imgs/mango.png'));
-          break;
-        case 4: setAvatar(require('../assets/imgs/dragonfruit.png'));
-          break;
-        case 5: setAvatar(require('../assets/imgs/lychee.png'));
-          break;
-        case 6: setAvatar(require('../assets/imgs/bokchoy.png'));
-          break;
-        default: setAvatar(require('../assets/imgs/placeholder.png'));
-      }
+  const [email, setEmail] = useState(currentUser.email);
+  const [phoneNum, setPhoneNum] = useState(currentUser.phoneNumber);
+  const [address, setAddress] = useState(currentUser.address);
 
-      const userEmailTwo = 'happyhippo@gmail.com';
-      setEmail(record[0].fields.email);
-      setAddress(record[0].fields.address);
-      setPhoneNum(record[0].fields['business phone']);
-    });
-  }, []);
+  const [avatar, setAvatar] = useState(placeholder);
 
-  const [isEditMode, setEditMode] = useState(false);
+  useEffect(() => {
+    switch (currentUser.avatarNum) {
+      case 1: setAvatar(pipa);
+        break;
+      case 2: setAvatar(eggplant);
+        break;
+      case 3: setAvatar(mango);
+        break;
+      case 4: setAvatar(dragonfruit);
+        break;
+      case 5: setAvatar(lychee);
+        break;
+      case 6: setAvatar(bokchoy);
+        break;
+      default: setAvatar(placeholder);
+    }
+  }, [refresh]);
 
-  function onEdit() {
+  const onEdit = () => {
     setEditMode(true);
     setTitle('Save');
-  }
+  };
 
-  function onSave() {
+  const onSave = async () => {
     if (email.length > 1 && phoneNum.length > 1 && address.length > 1) {
+      // TODO: add checker that the adjustments are different from what's already saved,
+      // that way users don't always have the alert
+      // TODO: add checker for email and phone number, can reuse checks from sign up screen
```

Comment:
Two implementations that are needed:
1. check if the changed inputs are even different from what was originally there, it's annoying to see the alert if nothing changes (this will likely require you to check currentUser's object fields and seeing if they are equal, search `currentUser` throughout the repo if you want to see examples)
2. sign up screen has several functions that check if the email is following a valid format or if the phone number has the right format or correct number of digits